node/service/src/lib.rs

```diff
@@ -231,47 +236,85 @@ impl IdentifyVariant for Box<dyn ChainSpec> {
 }
 
 pub fn frontier_database_dir(config: &Configuration, path: &str) -> std::path::PathBuf {
-	let config_dir = config
+	config
 		.base_path
-		.as_ref()
-		.map(|base_path| base_path.config_dir(config.chain_spec.id()))
-		.unwrap_or_else(|| {
-			BasePath::from_project("", "", "moonbeam").config_dir(config.chain_spec.id())
-		});
-	config_dir.join("frontier").join(path)
+		.config_dir(config.chain_spec.id())
+		.join("frontier")
+		.join(path)
```

Comment:
Just want to make sure this will result in the same thing. I think it will except I'm unclear about the previous `unwrap_or_else` part.

This is really important because we don't want nodes to implicitly start with a fresh db after upgrading :)